code/modules/vehicles/mecha/mech_fabricator.dm

```diff
@@ -10,6 +10,7 @@
 
 	subsystem_type = /datum/controller/subsystem/processing/fastprocess
 
+	interaction_flags_atom = parent_type::interaction_flags_atom | INTERACT_ATOM_MOUSEDROP_IGNORE_CHECKS
```

Comment:
```suggestion
	interaction_flags_atom = INTERACT_ATOM_MOUSEDROP_IGNORE_CHECKS
```

`NTERACT_ATOM_MOUSEDROP_IGNORE_CHECKS` overrides all values & doesn't care if we attach the parent type value or not